python/torch_mlir_e2e_test/test_suite/elementwise.py

```diff
@@ -949,28 +1078,34 @@ def forward(self, a):
 def ElementwiseReciprocalModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(4))
 
+
 # ==============================================================================
 
+
 class ElementwiseReciprocalIntModule(torch.nn.Module):
+
     def __init__(self):
         super().__init__()
+
     @export
     @annotate_args([
         None,
         ([-1], torch.int32, True),
     ])
-
     def forward(self, a):
         return torch.reciprocal(a)
 
 
 @register_test_case(module_factory=lambda: ElementwiseReciprocalIntModule())
 def ElementwiseReciprocalIntModule_basic(module, tu: TestUtils):
-    module.forward(torch.randint(1, 10, (4,), dtype=torch.int32))
+    module.forward(torch.randint(1, 10, (4, ), dtype=torch.int32))
```

Comment:
nit: for one element tuples in python, the convention I've always encountered is the one without the space